Makefile

```diff
@@ -0,0 +1,22 @@
+DOCKER := $(shell { command -v podman || command -v docker; })
```

Comment:
Doesn't podman automatically create a `docker` that just links to podman and runs it under the hood? At least on my Fedora machine it does. This seems maybe superfluous?

Reply:
I do not believe so (I just installed podman on my machine), I think some folks choose to alias docker to podman so they don't have to worry about it